lib/active_job/queue_adapters/resque_ext.rb

```diff
@@ -261,7 +261,7 @@ def requeue(job)
           resque_job = job.raw_data
           resque_job["retried_at"] = Time.now.strftime("%Y/%m/%d %H:%M:%S")
 
-          redis.lset(queue_redis_key, job.position, resque_job)
+          redis.lset(queue_redis_key, job.position, resque_job.to_json)
```

Comment:
this was the main change

Reply:
Making material changes to the library in a dev deps update is a sign the PR is overreaching. Use a separate PR for this.